test/LEAF_Request_Portal_Tests/tests/api/FormEditorControllerTest.php

```diff
@@ -897,4 +897,25 @@ public function testIndicatorPrivileges_setPrivileges_invalidInput() : void
         $this->assertNotNull($res);
         $this->assertFalse($res);
     }
+
+    /**
+     * Tests the `formEditor/[digit]/formParallelProcessing` endpoint.
+     */
+    public function testSetFormParallelProcessing() : void
+    {
+        $category = self::$client->get(array('a' => 'formStack/categoryList/all'))[1];
+        $this->assertNotNull($category);
+        $this->assertEquals('form_f4687', $category['categoryID']);
+        $this->assertEquals('1', $category['parallelProcessing']);
```

Comment:
Typo: should be `assertEquals('0', $category['parallelProcessing']);`

Reply:
default is 0 i'm enabling it